packages/riipen-ui/src/components/Editor.jsx

```diff
@@ -145,12 +150,19 @@ class Editor extends React.Component {
      * Optional style applied to parent div of editor.
      * Used to set a minimum height.
      */
-    style: PropTypes.object
+    style: PropTypes.object,
+
+    /**
+     * Additional action controls to display on the left side of the editor control bar.
+     */
+    stylingControls: PropTypes.arrayOf(PropTypes.node)
   };
 
   static defaultProps = {
-    additionalControls: [],
-    controlPosition: "top"
+    autoFocus: true,
```

Comment:
Should this default to true? If the editor is being displayed in a form and it is say the 3rd form element, I don't think we should auto focus it.

Reply:
its the current behaviour, I have no problem changing it, but this was mainly to prevent it from occurring on the messages page as it was stealing focus from the search input when the url changed

Reply:
Ya I think default to false, then we change the message page to use `autoFocus={true}`.